examples/advanced_operations/add_responsive_search_ad_full.py

```diff
@@ -0,0 +1,574 @@
+#!/usr/bin/env python
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+This example shows how to create a complete Responsive Search ad.
+
+Includes creation of: budget, campaign, ad group, ad group ad,
+keywords, geo targeting, and image extensions.
+
+More details on Responsive Search ads can be found here:
+https://support.google.com/google-ads/answer/7684791
+"""
+
+import argparse
+import sys
+import uuid
+import requests
+
+from google.ads.googleads.client import GoogleAdsClient
+from google.ads.googleads.errors import GoogleAdsException
+
+
+# Keywords from user.
+_KEYWORD_TEXT_EXACT_1 = "example of exact match"
```

Comment:
I'm a bit confused the naming of the variables. An initial underbar in Python is an indication of a variable that is only for internal use. An indication that this is a private variable. This is used primarily within a class or other limited scope.

https://peps.python.org/pep-0008/

_single_leading_underscore: weak “internal use” indicator. E.g. from M import * does not import objects whose names start with an underscore.

Reply:
I asked for these to be named this way. It's consistent with our other examples (see [here](https://github.com/googleads/google-ads-python/blob/main/examples/advanced_operations/add_smart_campaign.py#L31)), which do this to indicate that these vars are private. It's definitely possible that this style is incorrect, though my preference would be to maintain consistency in this PR and fix all of the examples at once in a different set of changes.

For leading underscores in the method names, I do think we actually removed those in the rest of the examples, so let's remove them here as well. @fblascogarma sorry for making that ask, I forgot we had made this style change.

Reply:
Marking this as resolved, Bob let's discuss internally whether we want to address this in all our examples.